modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/IgniteResource.java

```diff
@@ -32,6 +32,9 @@
     @Resources.BaseMessage("Cannot update field \"{0}\". You cannot update key, key fields or val field in case the val is a complex type.")
     Resources.ExInst<SqlValidatorException> cannotUpdateField(String field);
 
+    @Resources.BaseMessage("Cannot update field \"{0}\". Attempt to update primary keys partially.")
+    Resources.ExInst<SqlValidatorException> cannotUpdatePkPartially(String field);
```

Comment:
Because this particular patch is about MERGE command migration, and there is already separate [ticket](https://issues.apache.org/jira/browse/IGNITE-16529) for mentioned feature. Why do we need to merge both this tickets at once?

Reply:
Mentioned issue a little bit complicated than this one, here i restrict insertion of partially assembled PK.